lldb/tools/lldb-dap/DAP.cpp

```diff
@@ -778,28 +817,121 @@ llvm::Error DAP::Disconnect(bool terminateDebuggee) {
   return ToError(error);
 }
 
+bool DAP::IsCancelled(const protocol::Request &req) {
+  std::lock_guard<std::mutex> lock(m_cancelled_requests_mutex);
+  return m_cancelled_requests.contains(req.seq);
+}
+
+void DAP::ClearCancelRequest(const CancelArguments &args) {
+  std::lock_guard<std::mutex> cancalled_requests_lock(
```

Comment:
```suggestion
  std::lock_guard<std::mutex> lock(
```

It matches the name you use above, and I think the function is short enough..

Reply:
I noticed in a few different places the naming was 'guard' for the lock_guard variable. Swapped all my variables to 'guard' instead.